### PR TITLE
chore: add `@tarilabs` to the bindings package

### PR DESCRIPTION
**Pull request by Cifko**
Description
---
Add the `@tarilabs` prefix into the bindings package name.

### CODE REVIEW

File: applications/tari_dan_wallet_web_ui/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "react-dom": "^18.2.0",
     "react-icons": "^4.9.0",
     "react-router-dom": "^6.9.0",
-    "tari-bindings": "file:../../bindings",
+    "@tarilabs/tari-bindings": "file:../../bindings",
```
Comment (sdbondi) [inline applications/tari_dan_wallet_web_ui/package.json]: Maybe:
```suggestion
    "@tarilabs/typescript-bindings": "file:../../bindings",
```

**Comment by tarilabs** [comment]
Hi 👋 
I want to notify @Cifko @sdbondi every commit message where `@tarilabs` is not escaped/quoted, GitHub send me a notification 😅 🤣 

I believe you can notice that as the GitHub interface highlight and hyperlink for commit message containing `@validgithubhandle`

File: applications/tari_dan_wallet_web_ui/package.json (hunk 1 above)
Comment (Cifko) [reply applications/tari_dan_wallet_web_ui/package.json]: good idea, renamed everywhere

**Comment by sdbondi** [comment]
@tarilabs lol thanks for the heads up - sure will do :)

**Comment by Cifko** [comment]
> Hi 👋 I want to notify @Cifko @sdbondi every commit message where `@tarilabs` is not escaped/quoted, GitHub send me a notification 😅 🤣
> 
> I believe you can notice that as the GitHub interface highlight and hyperlink for commit message containing `@validgithubhandle`

sorry, should be fixed :-)

**Review by sdbondi — APPROVED** [review]

**Pull request merged**